README.md

```diff
@@ -370,6 +370,22 @@ $ remme batch get \
 }
 ```
 
+Get a batch status by its identifier — ``remme batch get-status``:
+
+| Arguments   | Type   |  Required | Description                       |
+| :---------: | :----: | :-------: | --------------------------------  |
+| id          | String |  Yes      | Identifier to get a batch status by.       |
```

Comment:
It is necessary that nothing climbs outside the table layout.

Reply:
Done.

Reply:
```suggestion
| id          | String |  Yes      | Identifier to get a batch status by. |
```